src/main/java/co/edu/etitc/sistemas/programacion/controller/BibliotecaController.java

```diff
@@ -0,0 +1,73 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package co.edu.etitc.sistemas.programacion.controller;
+
+import co.edu.etitc.sistemas.programacion.model.Computador;
+import co.edu.etitc.sistemas.programacion.model.Libro;
+import co.edu.etitc.sistemas.programacion.model.Periodico;
+import co.edu.etitc.sistemas.programacion.model.Recurso;
+import co.edu.etitc.sistemas.programacion.services.ServicioBiblioteca;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping({"/api/"})
+public class BibliotecaController {
+    @Autowired
+    private ServicioBiblioteca servicioBiblioteca;
+
+    public BibliotecaController() {
+    }
+
+    @GetMapping(
+            path = {"/recursos"},
+            produces = {"application/json"}
+    )
+    public List<Recurso> obtenerRecursos() {
+        return this.servicioBiblioteca.obtenerTodos();
+    }
+
+    @GetMapping(
+            path = {"/recursos/buscar"},
```

Comment:
En REST no ponemos acciones en la URI. Las acciones de HTTP hacen ese trabajo. No se necesita poner buscar, porque con el método GET de HTTP se entiende que es una búsqueda. La propuesta de cambio seria usar el siguiente endpoint: /recursos

```suggestion
            path = {"/recursos"},
```